### DISCUSSION TITLE
Slash command deadline

### DISCUSSION THREAD

**Discussion by okay1204** [nextcord/nextcord#166] [Q&A] [answered]
Do you guys think you'll be able to finish slash commands before April 2022?

**Comment by osp54** [comment]
🤣

**Comment by bensonchow123** [comment]
At this rate, I think this deadline is too optimistic.

**Comment by TAG-Epic** [comment, marked as answer]
Should probably be possible.

**Reply by okay1204** [reply]
🙏

**Comment by MoonlightCapital** [comment]
I was previously involved with them, but due to too many time constraints I'm passing the baton for the time being.

**Comment by Nopus64** [comment]
@TAG-Epic i saw you are working on slash-commands atm.

may that Helps. 😄 
https://github.com/goverfl0w/discord-interactions
